app/src/services/migrations/20191016164716_init.js

```diff
@@ -0,0 +1,48 @@
+
+exports.up = function (knex) {
```

Comment:
The up and down functions can be declared as anonymous () => {} functions

Reply:
that's what knex generates (./node_modules/.bin/knex migrate:make file) so that's what i am using.